doc/locale/ja/LC_MESSAGES/server.po

```diff
@@ -449,6 +449,13 @@ msgstr ""
 "`--protocol` オプションとその引数により、サーバのプロトコルを指定することがで"
 "きます。\"memcached\"はmemcachedバイナリプロトコルを示しています。"
 
+msgid ""
+"You don't need to create table.  When groonga receive request, it create "
```

Comment:
`table` is a countable noun. It need to add "a" here.
And `request` is also a countable. Ditto.